benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -937,7 +937,8 @@ void GraphicsBenchmarkApp::ProcessKnobs()
             SetupSpheres();
         }
         else {
-            const bool requireMoreSpheres = sphereInstanceCountKnobChanged && (pSphereInstanceCount->GetValue() > mInitializedSpheres);
+            const uint32_t initializedCount   = static_cast<uint32_t>(pSphereInstanceCount->GetValue());
+            const bool     requireMoreSpheres = sphereInstanceCountKnobChanged && (initializedCount > mInitializedSpheres);
```

Comment:
I wonder, do we need to check `sphereInstanceCountKnobChanged`?

Reply:
sphereInstanceCountKnobChanged is not the best name for it. It is "user has released the slider and values are final". I'll add this weirdness to the knob design update doc.

Reply:
ah, cool, thanks